src/Pages/StaffManagementPage/EmployeeManagement.jsx

```diff
@@ -15,16 +16,29 @@ function EmployeeManagement() {
     }, []);
 
     return (
-        <section className="wrapper">
-            <h1>Employee Management</h1>
-
-            {employees.map((employee) => (
-                <EmployeeComponent employee={employee} />
-            )
-            )}
-
-        </section>
-
+        <>
+            <div className="wrapper">
```

Comment:
Change div to main and place header outside of main. This should not affect the how the website looks